specify/specify7
# Stop cloning the `barcode` field

Fixes #3787

**Steps to test:**
1. Go to a record that has a barcode value on the preparation form
2. Clone the record
3. Verify that the barcode is not being cloned

Tested locally and it seems to do the job.
